cf_units/__init__.py

```diff
@@ -384,7 +384,22 @@ def suppress_errors():
                 'System', 'udunits2_xml_path',
                 default=os.path.join(sys.prefix, 'share', 'udunits',
                                      'udunits2.xml'))
-            _ud_system = _ut_read_xml(_alt_xml_path.encode())
+            if os.name == 'nt':
+                # On Windows systems we need to read the xml while on the
+                # system drive but we don't want to change the
```

Comment:
This assertion is actually incorrect: you need to read the XML while your CWD is **on the drive that the XML is located upon**.